Assets/MixedRealityToolkit/_Core/Interfaces/InputSystem/IPointer.cs

```diff
@@ -2,15 +2,11 @@
 // Licensed under the MIT License. See LICENSE in the project root for license information.
 
 using System.Collections;
-using Microsoft.MixedReality.Toolkit.InputSystem.Cursors;
-using Microsoft.MixedReality.Toolkit.InputSystem.Focus;
-using Microsoft.MixedReality.Toolkit.InputSystem.Gaze;
-using Microsoft.MixedReality.Toolkit.InputSystem.InputHandlers;
-using Microsoft.MixedReality.Toolkit.InputSystem.Sources;
-using Microsoft.MixedReality.Toolkit.Internal.Interfaces;
+using Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem.Handlers;
+using Microsoft.MixedReality.Toolkit.Internal.Utilities;
 using UnityEngine;
 
-namespace Microsoft.MixedReality.Toolkit.InputSystem.Pointers
+namespace Microsoft.MixedReality.Toolkit.Internal.Interfaces.InputSystem
 {
     public interface IPointer : IEqualityComparer
```

Comment:
MRTK Interface names should be prefixed with either IMixedReality or IMixedRealityToolkit

Reply:
Copy pasta done.